components/footer/footer.jsx

```diff
@@ -1,37 +1,89 @@
 import Link from 'next/link'
+import { useState } from 'react'
+import Image from 'next/image'
 
 import { Facebook, Instagram } from '../UI/svg'
 import s from './footer.module.scss'
 
 const year = new Date().getFullYear()
 
 export default function Footer() {
+  const [supportVisible, setSupportVisible] = useState(false)
+  const [discoverVisible, setDiscoverVisible] = useState(false)
+  const [infoVisible, setInfoVisible] = useState(false)
+
+  const isMobile = () => {
+    if (typeof window !== 'undefined') {
+      return window.innerWidth < 800
+    }
+    return false
+  }
+
+  const toggleSupport = () => {
+    if (isMobile()) {
+      setSupportVisible(!supportVisible)
+    }
+  }
+
+  const toggleDiscover = () => {
+    if (isMobile()) {
+      setDiscoverVisible(!discoverVisible)
+    }
+  }
+
+  const toggleInfo = () => {
+    if (isMobile()) {
+      setInfoVisible(!infoVisible)
+    }
+  }
+
   return (
     <footer>
       <div className={s.root}>
         <div className={s.support}>
-          <div className={s.category_name}>SUPPORT</div>
-          <Link href="/contact">Contact</Link>
-          <Link href="/faq">FAQ&apos;s</Link>
-          <Link href="/delivery">Delivery</Link>
-          <Link href="/returns">Return & Exchanges</Link>
-          <Link href="/legal">Terms & Conditions</Link>
-          <Link href="/privacy">Privacy Policy</Link>
+          <div className={s.category_name} onClick={toggleSupport}>
```

Comment:
It works, but for accessibility, this should be a button with the global class .resetButton

Reply:
example: https://github.com/Eloi-Perez/just-by-emma/blob/main/components/carousel/carousel.jsx